examples/thermo.jl

```diff
@@ -30,35 +33,44 @@ function µ_kinetic(I, km::KineticModel; Ω=Ω, muoA=muoA, muoB=muoB, T=T)
 end
 function g_kinetic(I, km::KineticModel; Ω=Ω, muoA=muoA, muoB=muoB, T=T)
     thermo_term(x) = g_thermo(x; Ω=Ω, muoA=muoA, muoB=muoB, T=T)
-    kinetic_term(x) = quadgk.(y->fit_overpotential((1 .- y) .* Ref(km), I), zero(x), x)
-    # quadgk always returns a tuple of (val, error) (or when we broadcast, an array of such tuples) so we need these two separate dispatches to get the return type to match the input type
-    g(x::AbstractVector) = thermo_term(x) .+ [k[1] for k in kinetic_term(x)]
-    g(x::Real) = thermo_term(x) + kinetic_term(x)[1]
+    function kinetic_term(x, w, n)
```

Comment:
this is the source of the CI failures – could we restore the behavior where feeding in a scalar gives a scalar instead of a single-element vector? That's the intuitive thing I would expect.